validator/rpc/apimiddleware/custom_hooks.go

```diff
@@ -17,7 +18,11 @@ func setVoluntaryExitEpoch(
 	req *http.Request,
 ) (apimiddleware.RunDefault, apimiddleware.ErrorJson) {
 	if _, ok := endpoint.PostRequest.(*SetVoluntaryExitRequestJson); ok {
-		j := &SetVoluntaryExitRequestJson{Epoch: req.URL.Query().Get("epoch")}
+		var epoch = req.URL.Query().Get("epoch")
+		if !regexp.MustCompile(`^[0-9]+?`).MatchString(epoch) {
```

Comment:
Checking the error returned from `strconv.ParseUint` is easier than writing a proper regex. This regex allows things like `000` or `123xyz`. Also I would prefer returning an error rather than silently defaulting to zero.